content/md/supplier-onboarder/handle-category.md

```diff
@@ -0,0 +1,102 @@
+---
+id: handle-category
+themes: manage-products
+title: Handle **categories**
+popular: false
+---
+
+# Definition of a category
+In Akeneo Onboarder, you can use **a category to classify products**. A category is always part of a category tree (or *classification tree*).
+
+As an Onboarder user, you can only have **one category tree** in Akeneo with an **unlimited number of levels** (categories, subcategories, sub-subcategories).
+
+![image of the category trees](../img/Settings_CategoryTree.png)
+
+## Differences between families and categories
+In this section, we define two principles of the Akeneo Onboarder: *Categories* and *Families*.
```

Comment:
```suggestion
In this section, we define two key concepts of the Akeneo Onboarder: *Categories* and *Families*.
```

Reply:
maybe think about this structure
- def cat
- def fam 
- difference cat Vs fam